app/jobs/virtual_hearings/create_conference_job.rb

```diff
@@ -81,7 +81,11 @@ def perform(hearing_id:, hearing_type:, email_type: :confirmation)
     create_conference unless virtual_hearing.active?
 
     # when a conference has been created and emails sent, the virtual hearing can be established
-    send_emails(email_type) if virtual_hearing.active?
+    begin
+      send_emails(email_type) if virtual_hearing.active?
+    rescue StandardError => error
+      capture_exception(error: error)
```

Comment:
I found this [error](https://sentry.prod.appeals.va.gov/department-of-veterans-affairs/caseflow/issues/15103/?referrer=webhooks_plugin) that appeared in #appeals-app-alerts where it shows hearing_id and hearing_type. I could not trace where that is added on but could we also add email_type and i think the following could help with that.
```suggestion
      capture_exception(error: error, extra: { virtual_hearing_id: virtual_hearing.id, email_type: email_type })
```

Reply:
good call! added in dd3209e